Sources/BraintreeShopperInsights/BTShopperInsightsClient.swift

```diff
@@ -29,7 +29,9 @@ public class BTShopperInsightsClient {
     /// - Parameters:
     ///   - request: A `BTShopperInsightsRequest` containing the buyer's user information
     /// - Returns: A `BTShopperInsightsResult` instance
-    /// - Note: This feature is in beta. It's public API may change or be removed in future releases.
+    /// - Note: This feature is in beta. Its public API may change or be removed in future releases.
+    ///         Shopper Insights for PayPal payment recommendation is only available in US and AU, FR, DE, ITA, NED, ESP, Switzerland and UK.
+    ///         Venmo recommendation is only available for US.
```

Comment:
Nit/take it or leave it -

```suggestion
    ///         PayPal recommendation is only available for US, AU, FR, DE, ITA, NED, ESP, Switzerland, and UK merchants
    ///         Venmo recommendation is only available for US merchants.
```